fact_plots/scripts/plot_theta_squared.py

```diff
@@ -117,22 +117,26 @@ def main(data_path, threshold, theta2_cut, key, bins, alpha, start, end, prelimi
 
     del events
 
-    limits = [0, 0.3]
```

Comment:
Is max_theta2 = 0.3 a result of trigonometry, 5 + 1 wobble positions and a wobble offset of 0.6 degree? 
I mean, does 0.3 reflect the radius of a circle around a wobble position, so no circles overlap?

Reply:
No, this is just what turned out to look nice.

Reply:
Wobble distance of 0.6 degrees would mean that the maximum theta2 without overlap would be 0.36 deg²

Reply:
Ah, no. wrong. It's smaller.

Reply:
If this is the case? Can the limit be calculated automatically from the wobble offset and the number of source positions?
If not could you ad a comment telling what this number means

Reply:
Why? What has the maximum theta we show in a theta square plot something to do with that?

Reply:
FYI, the maximum theta² cut you can do without overlapping regions should be

(wobble_distance * sin(π / (n_off + 1)))²


Which is 0.89 for 5 off positions and 0.6° wobble distance

Reply:
But again, this has imho nothing to do with the xrange of the theta2 plot.

Reply:
Ok, thanks for the clarification. I was just stumbling over this number and was wondering if it has a deeper meaning ;-). If it does not, leaving a comment what come handy to not misunderstand it, but is not mandatory imho.

Reply:
I missed a 0, which is pretty significant in this case ;-)

Reply:
I was mislead because (wobble_distance * sin(π / (n_off + 1))) = 0.3, with n_off=5